fontforge/python.c

```diff
@@ -293,15 +293,20 @@ static PyFF_Layer *LayerFromLayer(Layer *,PyFF_Layer *);
 /* ************************************************************************** */
 /* Find python objects corresponding to non-python structures */
 /* ************************************************************************** */
-static PyFF_Font* PyFF_FontForFV( FontViewBase *fv ) {
+PyObject* PyFF_FontForFV( FontViewBase *fv ) {
     if ( fv==NULL )
 	return NULL;
+    if ( fv->python_fv_object==NULL ) {
+        fv->python_fv_object = PyFF_FontType.tp_alloc(&PyFF_FontType,0);
+        ((PyFF_Font *) (fv->python_fv_object))->fv = fv;
+    }
+    Py_INCREF( (PyObject *) (fv->python_fv_object) );
```

Comment:
Also, if you're going to take this approach you can't just move the increment, you'll have to increment *twice* on the creation path, once inside the NULL guard and once outside. Otherwise you'll be down a reference when the caller is expecting an increment. 

This is part of why the traditional approach may make more sense. Reference counting can get hairy.